test/boost/mutation_reader_test.cc

```diff
@@ -2134,6 +2134,9 @@ SEASTAR_THREAD_TEST_CASE(test_foreign_reader_destroyed_with_pending_read_ahead)
             return make_ready_future<std::tuple<control_type, reader_type>>(std::tuple(std::move(control), std::move(reader)));
         }).get0();
 
+        auto& remote_control = std::get<0>(remote_control_remote_reader);
+        auto& remote_reader = std::get<1>(remote_control_remote_reader);
```

Comment:
Shouldn't it be auto instead of auto& ?

Reply:
No, why?

Reply:
Previously remote_control and remote_reader were lvalues and now they are lvalue references. Maybe it's ok to change it as well. I was just pointing out the place that, I believe, has changed.

Reply:
The were not variables, they were structured bindings.

`remote_control_remote_reader` is a variable in the same scope, so the references don't change the lifetime.